src/background/js/environment.js

```diff
@@ -1 +1 @@
-var ENV = "production";
```

Comment:
again, pay attention to your diffs before you push stuff to your remote branch. This file should not be committed.

Reply:
That file is already in the .gitignore. Then how is it added to the commit?
Should i run these before i push?
```
git add -u
git reset -- src/backgrount/js/environment.js
```